src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs

```diff
@@ -90,7 +90,7 @@ internal Regex(string pattern, RegexOptions options, TimeSpan matchTimeout, Cult
             if ((options & RegexOptions.NonBacktracking) != 0)
             {
                 // If we're in non-backtracking mode, create the appropriate factory.
-                factory = new SymbolicRegexRunnerFactory(tree, options, matchTimeout, culture);
+                factory = new SymbolicRegexRunnerFactory(tree, options, matchTimeout);
```

Comment:
There was some IgnoreCase stuff in this implementation, and none of that is relevant anymore (the only place IgnoreCase exists in the RegexNode tree after parsing is with Backreferences, and SymbolicRegexRunnerFactory doesn't support backreferences).